docs/en/development.rst

```diff
@@ -188,8 +188,8 @@ templates that bake uses. The best way to do this is:
 
 #. Bake a new plugin. The name of the plugin is the bake 'theme' name
 #. Create a new directory **plugins/[name]/src/Template/Bake/Template/**.
```

Comment:
this line forgotten

Reply:
how would it be correct?

Reply:
Wouldn't the path be `plugins/[name]/templates/bake/`?